src/helpers.coffee

```diff
@@ -31,13 +32,16 @@ jade = async ({source, target, data}) ->
   render = _jade.compile source.content, filename: source.path
   target.content = render data
 
+handlebars = async ({source, target, data}) ->
+  source.content ?= yield read source.path
+  target.content = _handlebars source.content, data
```

Comment:
You're correct about the implementation. I didn't realize I was in the P9K repo. Doh.

Helpers currently only return promises when the underlying implementation is async. So you're correct about that. Since these are helper functions, and not part of an interface, there's no need to normalize the return value.